functests/1_performance/cpu_management.go

```diff
@@ -83,7 +83,9 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			By(fmt.Sprintf("Allocatable CPU should be less then capacity by %d", len(listReservedCPU)))
 			capacityCPU, _ := workerRTNode.Status.Capacity.Cpu().AsInt64()
 			allocatableCPU, _ := workerRTNode.Status.Allocatable.Cpu().AsInt64()
-			Expect(capacityCPU - allocatableCPU).To(Equal(int64(len(listReservedCPU))))
+			differenceCPUGot := capacityCPU - allocatableCPU
+			differenceCPUExpected := int64(len(listReservedCPU))
+			Expect(differenceCPUGot).To(Equal(differenceCPUExpected), "Allocatable CPU %d should be less then capacity %d by %d - got %d instead", allocatableCPU, capacityCPU, differenceCPUExpected, differenceCPUGot)
```

Comment:
Nit: the sentence's punctuation wasn't so clear to me.
I would replace the ```-``` with ```,``` or ```.``` , thought it's a minus sign at first glance. 
Also, I think it should be ```less than``` instead of ```less then```

Reply:
done